nmdc_runtime/site/translation/neon_surface_water_translator.py

```diff
@@ -29,13 +29,17 @@
 SURFACE_WATER_LOCAL_SCALE_MAPPINGS = {
     "lake": {
         "buoy.c0": {"term_id": "ENVO:01001191", "term_name": "water surface"},
+        "buoy.b.c0": {"term_id": "ENVO:01001191", "term_name": "water surface"},
         "buoy.c1": {"term_id": "ENVO:00002131", "term_name": "epilimnion"},
         "buoy.c2": {"term_id": "ENVO:00002269", "term_name": "thermocline"},
         "buoy.c3": {"term_id": "ENVO:00002130", "term_name": "hypolimnion"},
         "littoral": {
             "term_id": "ENVO:01000409",
             "term_name": "freshwater littoral zone",
         },
+        "S2": {"term_id": "ENVO:00002132", "term_name": "metalimnion"},
+        "reach": {"term_id": "ENVO:01001191", "term_name": "water surface"},
+        "inflow": {"term_id": "ENVO:01001191", "term_name": "water surface"},
```

Comment:
@aclum: there are some new term values that need a fresh "classification". I've identified some of them here and assigned a general term_id and term_name for now.

Reply:
@sujaypatil96  I dont think we need to add any of these new term values. `buoy.b.c0`, `S2` and `reach` refer to river or streams and we can use the existing logic on lines 44-45. I don't see inflow in any of the provisional release data. Do you have an example sample name where that is used and what release that came from?

Reply:
@aclum yes, here's an example of a NEON sample that has `inflow` in it's `env_local_scale`.

Biosample name: `TOOK.IF.20220702`
Aquatic site type: `lake`
Named location: `TOOK.AOS.inflow`

Reply:
We are downloading the "expanded" release and not the "provisional" release of the data products.

Reply:
Odd i must not have downloaded the providential data. Please use http://purl.obolibrary.org/obo/ENVO_00000476 for inflow

Reply:
Great! thanks @aclum. I've made changes to the translator to reflect the above comments, i.e., remove `buoy.b.c0`, `S2` and `reach`, and also add a mapping for `lake` / `inflow`.